Directory.Build.targets

```diff
@@ -1,7 +1,3 @@
 <Project>
-  <ItemGroup>
```

Comment:
just to explain this change - I moved it away so we will not enable StyleCop for Thrift and Samples